docs/guides/burst_insar_product_guide.md

```diff
@@ -235,6 +259,8 @@ The tags and extensions used and example file names for each raster are listed i
 
 {% set base_name = 'S1<wbr>_136231<wbr>_IW2<wbr>_20200604<wbr>_20200616<wbr>_VV<wbr>_INT80<wbr>_12E3<wbr>' %}
 
```

Comment:
When we transition to the multi-burst job only, the base name should be changed to the convention used for multi-burst